bin/fetchdata.js

```diff
@@ -15,6 +15,18 @@ axiosRetry(axios, {
     retryDelay: () => (Math.random() * 2000) + 1000      // jitter retry delay by 1 - 3 seconds
 });
 
+function populateMissingData(attributes, id) {
+    if (id === '3941784506' || id === '3463348370') {
+        attributes.type = {
+            data: {
+                attributes: {
+                    name: 'token unit'
```

Comment:
Just checking, did you confirm that the generated jsons in the test folder look correct?